packages/web-components/src/label/label.template.ts

```diff
@@ -0,0 +1,13 @@
+import { html } from '@microsoft/fast-element';
+import { Label } from './label.js';
+
+/**
+ * The template for the Fluent label web-component.
+ * @public
+ */
+export const template = html`
+  <template>
```

Comment:
Last thing, since you don't need to set anything on the host binding target (`<template>`) the `<template>` tag is not needed here.